server/schedule/operator/operator.go

```diff
@@ -209,6 +209,9 @@ func (o *Operator) CheckTimeout() bool {
 
 // Len returns the operator's steps count.
 func (o *Operator) Len() int {
+	if o == nil {
```

Comment:
When will this happen? I think better to update test to check nil.

Reply:
Done.